src/main/scala/beam/agentsim/agents/PersonAgent.scala

```diff
@@ -203,6 +203,43 @@ class PersonAgent(
     else Right(_experiencedBeamPlan.activities(ind))
   }
 
+  def getAverageTravelTime(linkTravelTime: IndexedSeq[Int]):IndexedSeq[Int] = {
+    if (linkTravelTime.isEmpty) Vector()
+    else {
+      // FIXME There are the way to make it faster and even lazy
+      // We don't need the travel time for the last link, so we drop it (dropRight(1))
```

Comment:
If I looked at a method "getAverageTravelTime" I would assume that I'd get the travel times for all links, not with the last one dropped. If the caller of getAverageTravelTime doesn't need the last time, then the *caller* should do the dropping, not the method.

Reply:
Hmm, but why are we averaging in the first place?